galaxy_importer/main.py

```diff
@@ -50,6 +48,26 @@ def main(args=None):
     write_output_file(data)
 
 
+def setup_logger(cfg):
+    """Sets up logger with custom formatter."""
+    logger = logging.getLogger()
+    logger.setLevel(getattr(logging, cfg.log_level_main, 'INFO'))
+
+    ch = logging.StreamHandler(stream=sys.stdout)
+    ch.setFormatter(CustomFormatter())
+    logger.addHandler(ch)
+
+
+class CustomFormatter(logging.Formatter):
+    """Formatter that does not display INFO loglevel."""
```

Comment:
Not sure I understand the requirement for not showing the log levelname when it is INFO?  Is it to preserve output format with something for compatibility?

Reply:
Not super important, but since we're displaying ansible-test sanity stdout, it just prevents lines that look like `INFO: ERROR: <some sanity error>`